### DISCUSSION TITLE
Use create-vue instead of Vue CLI?

### DISCUSSION THREAD

**Discussion by felix-bohlin** [vuetifyjs/vuetify#15772] [General] [answered]
Since Vue CLI is in maintenance mode and they recommend create-vue instead for scaffolding projects, will this be the plan going forwards with Vuetify? 

Just curious. Cheers.

**Comment by KaelWD** [comment, marked as answer]
vue-cli-plugin-vuetify currently still works with projects made by create-vue, and we've started our own: https://github.com/vuetifyjs/create-vuetify
Or you can use create-vue and set vuetify up yourself, it isn't that complicated.

**Reply by felix-bohlin** [reply]
Thanks for the quick reply Kael 👍 

I know the documentation isn't finished yet, it's just that the "Get started" guide explicitly says "Vuetify 3 **requires** Vue CLI 5.0 in order to install properly" so that's why I asked.

**Reply by KaelWD** [reply]
Yeah that's not worded super well lol, if you're using vue-cli it has to be v5 so you get webpack 5. And even then I think a basic setup is possible with webpack 4 and babel, you just won't be able to use webpack-plugin-vuetify.
